### PR TITLE
added old files for adapterbox

### PR DESCRIPTION
**Pull request by dakhnod**
Opened because of https://xentral.atlassian.net/browse/XEN-2144

Adds old script and .deb package, since new script seems not to be available yet.

### CODE REVIEW

**Comment by dakhnod** [comment]
Also, fixes from this (https://xentral.atlassian.net/browse/XEN-2055) ticket will be pushed as soon as they are available.

File: adapterbox/install.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+sudo apt-get --yes install libcurl3 php7.0-curl php7.0-cli php7.0-xml
```
Comment (bambamboole) [inline adapterbox/install.sh:1]: 7.0 is more than old. I think a simply only put 7 instead of 7.0 would be enough

Comment (dakhnod) [reply adapterbox/install.sh:1]: I would do that when fixing the whole script. The scope now is just to make everything available as it is.

File: adapterbox/install.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+sudo usermod -a -G lpadmin pi
+sudo systemctl enable ssh
+wget http://www.wawision.de/adapterbox.deb 
+sudo dpkg -i adapterbox.deb
+echo "Seriennummer eingeben gefolgt von einem [ENTER]:"
+read seriennummer
+echo "<?php \$config['serial'] = '$seriennummer'; ?>" > /tmp/config.php
+sudo mv /tmp/config.php /root/config.php
```
Comment (tchabomep) [inline adapterbox/install.sh:9]:
```suggestion
sudo mv -f /tmp/config.php /root/config.php
```
In case the file this should override it

**Review by tchabomep — COMMENTED** [review]
Please check my suggestion

File: adapterbox/install.sh (hunk 2 above)
Comment (dakhnod) [reply adapterbox/install.sh:9]: this PR should only make the old files available. Improving the code will be another PR.

**Review by gassnerli — APPROVED** [review]

**Review by tchabomep — APPROVED** [review]

**Pull request merged**